src/extensions/default/BrambleUrlCodeHints/camera/video.js

```diff
@@ -69,7 +69,10 @@ define(function (require, exports, module) {
         vendorURL.revokeObjectURL(this.interface.src);
 
         if(this.stream) {
-            this.stream.stop();
+            var video = this.stream.getVideoTracks()[0];
```

Comment:
Let's not have it throw here in case this doesn't exist in some browser:

```var track = this.stream.getVideoTracks ? this.stream.getVideoTracks()[0] : null;```